notifier.rst

```diff
@@ -0,0 +1,562 @@
+.. index::
+   single: Notifier
+
+Sending Notifications with Notifier
+===================================
+
+.. versionadded:: 5.0
+
+    The Notifier component was introduced in Symfony 5.0 as an
+    :doc:`experimental feature </contributing/code/experimental>`.
+
+Current web applications use many different channels to send messages to
+the users (e.g. SMS, Slack messages, emails, push notifications, etc.). The
+Notifier component in Symfony is an abstraction on top of all these
+channels. It provides a dynamic way to manage how the messages are send.
+Get the Notifier installed using:
+
+.. code-block:: terminal
+
+    $ composer require symfony/notifier
+
+Channels: Chatters, Texters, Email and Browser
+----------------------------------------------
+
+The notifier component can send notifications to different channels. Each
+channel can integrate with different providers (e.g. Slack or Twilio SMS)
+by using transports.
+
+The notifier component supports the following channels:
+
+* `SMS <SMS Channel>`_ for simple text based notifications;
```

Comment:
Not sure about the descriptions of SMS and Chat ... it's not about simple vs advanced messages. What about this:

```rst
* `SMS <SMS Channel>`_, sends notifications to phones via SMS messages;
* `Chat <Chat Channel>`_ sends notifications to chat services like Slack and Telegram;
* `Email <Email Channel>`_ sends notifications via email using the :doc:`Symfony Mailer </mailer>`;
* Browser sends notifications to the browser using :ref:`flash messages <flash-messages>`.
```